code/style.css

```diff
@@ -1,4 +1,242 @@
 html {
-  background: #0025ff;
```

Comment:
Consider removing the first two lines. You already solved what you wanted on the lines below. :)